api/src/main/java/au/org/aodn/nrmn/restapi/validation/BaseFormattedValidator.java

```diff
@@ -12,5 +12,5 @@ public BaseFormattedValidator(String columnTarget) {
         this.columnTarget = columnTarget;
     }
 
-    abstract public <T> Validated<StagedRowError, T> valid(StagedRowFormatted target);
+    abstract public  Validated<StagedRowError, String> valid(StagedRowFormatted target);
```

Comment:
unneeded space between public and Validated

Reply:
space out